tools/parse_baselines/main.py

```diff
@@ -129,7 +139,7 @@ def main():
 
     # Finally, parse and update the baselines.
     baselines_path = Path(
-        f"./tests/integration_tests/performance/configs/test_{args.test}_config_{args.kernel}.json"
+        f"./tests/integration_tests/performance/configs/{OUTPUT_FILENAMES[args.test][0]}_config_{args.kernel}.json"
```

Comment:
We could also just delete the OUTPUT_FILENAMES dict too and depend on the convention that it is always `test_{test_name}...`

Reply:
This is actually the main issue that this PR intends to solve. We currently use `snapshot_restore_performance` as test name, but the output config files that should result from the parser are called `snap_restore_performance`. What I can do to consider your point of view (which I agree with actually) is to keep this convention and also rename those config files from `test_snap_restore_performance_config...` to `test_snapshot_restore_performance_config...`  that we currently have and then we can get rid of both dict you mentioned. However, we should then stick to this convention of naming, but this is not a bad thing overall. Hope I modified everywhere needed not to break smth.

Reply:
aah I see the issue now... `snap` vs `snapshot`. I agree.